packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/risk-management.tsx

```diff
@@ -1,27 +1,103 @@
 import React from 'react';
 import clsx from 'clsx';
-import { TextField } from '@deriv-com/quill-ui';
-import { Localize, localize } from '@deriv/translations';
+import { observer } from 'mobx-react';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { ActionSheet, TextField } from '@deriv-com/quill-ui';
+import { localize, Localize } from '@deriv/translations';
+import { getCurrencyDisplayCode } from '@deriv/shared';
+import Carousel from 'AppV2/Components/Carousel';
+import CarouselHeader from 'AppV2/Components/Carousel/carousel-header';
+import TradeParamDefinition from 'AppV2/Components/TradeParamDefinition';
+import { addUnit, isSmallScreen } from 'AppV2/Utils/trade-params-utils';
+import RiskManagementPicker from './risk-management-picker';
+import RiskManagementContent from './risk-management-content';
 
 type TRiskManagementProps = {
     is_minimized?: boolean;
 };
 
-const RiskManagement = ({ is_minimized }: TRiskManagementProps) => {
-    return (
-        <TextField
-            variant='fill'
-            readOnly
-            label={
-                <Localize
-                    i18n_default_text='Risk Management'
-                    key={`risk-management${is_minimized ? '-minimized' : ''}`}
+const RiskManagement = observer(({ is_minimized }: TRiskManagementProps) => {
+    const [is_open, setIsOpen] = React.useState(false);
+    const {
+        cancellation_range_list,
+        cancellation_duration,
+        currency,
+        has_cancellation,
+        has_take_profit,
+        has_stop_loss,
+        take_profit,
+        stop_loss,
+    } = useTraderStore();
+
+    const closeActionSheet = () => setIsOpen(false);
+    const getRiskManagementText = () => {
+        if (has_cancellation) return `DC: ${addUnit({ value: cancellation_duration, unit: localize('minutes') })}`;
```

Comment:
can it be 1 `minute`?

Reply:
localize not needed?

Reply:
The value is predefined and coming from BE. Minimum is 5 minutes. I can add check for 1 minute just in case, what do you think? It might be safer. 
<img width="395" alt="Screenshot 2024-08-26 at 10 05 58 AM" src="https://github.com/user-attachments/assets/257745d1-86e1-4890-bba4-1843e618bf29">

Reply:
Regarding localize: we need to wrap only unit with localize, because the value itself is coming from BE.